src/generator/convenience/client.ts

```diff
@@ -7,7 +7,7 @@ import { Session } from '@azure-tools/autorest-extension-base';
 import { camelCase } from '@azure-tools/codegen';
 import { CodeModel, Parameter } from '@azure-tools/codemodel';
 import { values } from '@azure-tools/linq';
-import { ContentPreamble, ImportManager, ParamInfo } from '../common/helpers';
+import { ContentPreamble, formatParamInfoTypeName, ImportManager, ParamInfo, sortParamInfoByRequired } from '../common/helpers';
```

Comment:
So this is a format convention question for me to keep in mind. Are we leaving functions uncapitalized? and interfaces/classes etc capitalized?

Reply:
I asked Garrett about this, his response was.
> type/interface/class names are PascalCased
variables, methods, properties, functions are camelCased

So I've been trying this (sorry I forgot to bring this up earlier).

Reply:
Ah ok good to know! Thanks for the info